packages/wdio-utils/tests/__mocks__/wdio-scoped-service.ts

```diff
@@ -1,4 +1,5 @@
 export default class MyService {
+    isScoped: boolean
```

Comment:
```suggestion
    isScoped = false
```

Then remove the constructor.
This can be done for similar classes.